aws/resource_aws_elasticsearch_domain_test.go

```diff
@@ -144,6 +144,36 @@ func TestAccAWSElasticSearchDomain_tags(t *testing.T) {
 	})
 }
 
+func TestAccAWSElasticSearchDomain_update(t *testing.T) {
+	var input elasticsearch.ElasticsearchDomainStatus
+
+	ri := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckESDomainDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccESDomainConfig_ClusterUpdate(ri),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckESDomainExists("aws_elasticsearch_domain.example", &input),
+					testAccCheckESNumberOfInstances(ri, input.ElasticsearchClusterConfig),
+				),
+			},
```

Comment:
Thanks! I've added [some code](https://github.com/DanKans/terraform-provider-aws/blob/eaa979576213d81369b4e2954070c71dd072254c/aws/resource_aws_elasticsearch_domain_test.go#L147). Could you please take a look and let me know what do you think?